server/app/auth/oidc/IdTokens.java

```diff
@@ -0,0 +1,68 @@
+package auth.oidc;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.nimbusds.jwt.JWT;
+import com.nimbusds.jwt.JWTClaimsSet;
+import com.nimbusds.jwt.JWTParser;
+import java.text.ParseException;
+import java.time.Clock;
+import java.util.Date;
+import java.util.Optional;
+
+/** Class that manages stored ID tokens for an Account. */
```

Comment:
**suggestion (blocking):** Please add more context in the javadoc for what an ID token is and why we store them

Reply:
Done